debian/control

```diff
@@ -35,6 +35,15 @@ Vcs-Browser: https://github.com/canonical/ubuntu-pro-client
 Rules-Requires-Root: no
 
 Package: ubuntu-advantage-tools
+Depends: ubuntu-pro-client (= ${binary:Version}), ${misc:Depends}
```

Comment:
This Depends together with `Architecture: all` (which is correct for a transitional package) causes the following lintian error:

```
E: ubuntu-advantage-tools source: not-binnmuable-all-depends-any ubuntu-advantage-tools -> ubuntu-pro-client
```

(Run `lintian-explain-tags not-binnmuable-all-depends-any` for more info on it.) Now, this is not an issue on Ubuntu, because in Ubuntu we don't name binNMUs, however I think we should still avoid in in any case.

The easiest way in my opinion is to drop the versioned dependency altogether, we don't need to worry about an ubuntu-pro-client package being "old" because we're just introducing it. It will be the "real" package to have versioned dependencies as needed.